test/CadlRanchMockApis/src/FirstTest-TypeSpec.ts

```diff
@@ -42,59 +42,3 @@ Scenarios.FirstTest_CreateLiteral = passOnSuccess([
     };
   }),
 ]);
```

Comment:
any reason why we have to keep this scenario here?

Reply:
fixed